Plane: Recommend not using autotune roll for RUDDER_ONLY planes (#6787)
ArduPilot/ardupilot_wiki: plane/source/docs/automatic-tuning-with-autotune.rst

The :ref:`AUTOTUNE_AXES<AUTOTUNE_AXES>` bitmask selects which axes will be tuned while in Autotune. Default is roll, pitch and yaw.

> .. warning:: Autotune is not designed to work well with :ref:`RUDDER_ONLY<RUDDER_ONLY>` vehicles for tuning roll. Consider only using autotune for the pitch axis and manually tuning roll and yaw.

Reviewer: ```suggestion
.. warning:: AUTOTUNE is not designed to work well with :ref:`RUDDER_ONLY<RUDDER_ONLY>` vehicles for tuning roll. :ref:`Manually tune the roll <new-roll-pitch-tuning>` axis instead.
```
I don't see why ACRO yaw autotuning wont work on rudder only planes...ailerons are not involved
